core/src/main/java/org/apache/hop/core/plugins/BasePluginType.java

```diff
@@ -79,16 +79,16 @@
 
   protected boolean searchLibDir;
 
-  protected Class<T> pluginType;
+  private Class<T> pluginClass;
```

Comment:
Note this is not a change related to sonarqube reported issue. I did not see any reference for this variable being outside this class.